cypress/integration/api/alerts/device_alerts.ts

```diff
@@ -0,0 +1,137 @@
+/// <reference path="../../../support/index.d.ts" />
+import moment from "moment";
+import { EventTypes } from "../../../commands/api/events";
+import { checkResponse, getCreds } from "../../../commands/server";
+import { getTestName } from "../../../commands/names";
+import { ComparableAlert, getExpectedAlert } from "../../../commands/api/alerts";
+import { ComparableEvent, getExpectedEvent } from "../../../commands/api/events";
+
+const AuthorizationError = 403;
+const BadRequest = 400;
+
+const OK = null;
+
+
+describe("Devices alerts", () => {
+  const group = "colonelGroup";
+  const user = "colonel";
+  const helper = 'frank';
+  const camera = 'camera1';
+  const conditions = [{"tag":"possum","automatic":true}];
+  const alertName = "alert1";
+  var oneAlert;
+  var emptyAlert;
+  var expectedEvent;
+  var alertId;
+  var recId;
+
+  before(() => {
+    cy.apiCreateUser(helper);
+    cy.apiCreateUserGroupAndCamera(user, group, camera);
+  });
+
+  it("Cannot create alert without access permissions", () => {
+    cy.apiAddAlert(helper,alertName,conditions,camera,null,AuthorizationError).then((response) => {checkResponse(response,AuthorizationError)});
+  });
+
+  it("Cannot create alert with invalid condition", () => { 
+    const bad_conditions = [{"tag": "any", "automaticF": true}]
+    cy.apiAddAlert(user,alertName,bad_conditions,camera,null,BadRequest).then((response) => {checkResponse(response,BadRequest)});
+  });
+
+  it("Can create alert and has no events by default", () => {
+    // create alert
+    cy.apiAddAlert(user,alertName,conditions,camera,0,200).then((response)=> {
+
+    // crete an example alert to compare against
```

Comment:
I can see why this is in the response - so that the ids actually exists.   Another way is to pass the user as the name, eg user: user and in the  apiCheckAlert then convert it to and id using getCreds.id.  

Also if we always compare all fields every time then if the api changes then each test will change.   We probably only need max one test that actually checks the user is correct. (Actually probably none as the apiCheckAlert might not find the alert if the user is wrong.